h/zssLogging.h

```diff
@@ -13,13 +13,570 @@
+#define ZSS_LOG_NO_LOAD_JWT_MSG              ZSS_LOG_NO_LOAD_JWT_MSG_ID" "ZSS_LOG_NO_LOAD_JWT_MSG_TEXT
+
+#ifndef ZSS_LOG_PATH_TO_SERVER_MSG_ID
+#define ZSS_LOG_PATH_TO_SERVER_MSG_ID        LOG_MSG_PRFX"1031W"
+#endif
+#define ZSS_LOG_PATH_TO_SERVER_MSG_TEXT      "Usage: zssServer <path to server.json file>\n"
+#define ZSS_LOG_PATH_TO_SERVER_MSG           ZSS_LOG_PATH_TO_SERVER_MSG_ID" "ZSS_LOG_PATH_TO_SERVER_MSG_TEXT
+
+#ifndef ZSS_LOG_SIG_IGNORE_MSG_ID
+#define ZSS_LOG_SIG_IGNORE_MSG_ID            LOG_MSG_PRFX"1032W"
+#endif
+#define ZSS_LOG_SIG_IGNORE_MSG_TEXT          "warning: sigignore(SIGPIPE)=%d, errno=%d\n"
+#define ZSS_LOG_SIG_IGNORE_MSG               ZSS_LOG_SIG_IGNORE_MSG_ID" "ZSS_LOG_SIG_IGNORE_MSG_TEXT
+
+#ifndef ZSS_LOG_SERVER_CONFIG_MSG_ID
+#define ZSS_LOG_SERVER_CONFIG_MSG_ID         LOG_MSG_PRFX"1033I"
+#endif
+#define ZSS_LOG_SERVER_CONFIG_MSG_TEXT       "Server config file=%s\n"
+#define ZSS_LOG_SERVER_CONFIG_MSG            ZSS_LOG_SERVER_CONFIG_MSG_ID" "ZSS_LOG_SERVER_CONFIG_MSG_TEXT
+
+#ifndef ZSS_LOG_SERVER_STARTUP_MSG_ID
+#define ZSS_LOG_SERVER_STARTUP_MSG_ID        LOG_MSG_PRFX"1034W"
+#endif
+#define ZSS_LOG_SERVER_STARTUP_MSG_TEXT      "server startup problem, address %s not valid\n"
+#define ZSS_LOG_SERVER_STARTUP_MSG           ZSS_LOG_SERVER_STARTUP_MSG_ID" "ZSS_LOG_SERVER_STARTUP_MSG_TEXT
+
+#ifndef ZSS_LOG_ZSS_SETTINGS_MSG_ID
+#define ZSS_LOG_ZSS_SETTINGS_MSG_ID          LOG_MSG_PRFX"1035I"
+#endif
+#define ZSS_LOG_ZSS_SETTINGS_MSG_TEXT        "ZSS server settings: address=%s, port=%d\n"
+#define ZSS_LOG_ZSS_SETTINGS_MSG             ZSS_LOG_ZSS_SETTINGS_MSG_ID" "ZSS_LOG_ZSS_SETTINGS_MSG_TEXT
+
+#ifndef ZSS_LOG_ZSS_STARTUP_MSG_ID
+#define ZSS_LOG_ZSS_STARTUP_MSG_ID           LOG_MSG_PRFX"1036W"
+#endif
+#define ZSS_LOG_ZSS_STARTUP_MSG_TEXT         "Server startup problem ret=%d reason=0x%x\n"
+#define ZSS_LOG_ZSS_STARTUP_MSG              ZSS_LOG_ZSS_STARTUP_MSG_ID" "ZSS_LOG_ZSS_STARTUP_MSG_TEXT
+
+#ifndef ZSS_LOG_PORT_OCCUP_MSG_ID
+#define ZSS_LOG_PORT_OCCUP_MSG_ID            LOG_MSG_PRFX"1037W"
+#endif
+#define ZSS_LOG_PORT_OCCUP_MSG_TEXT          "This is usually due to the server port (%d) already being occupied. Is ZSS running twice?\n"
+#define ZSS_LOG_PORT_OCCUP_MSG               ZSS_LOG_PORT_OCCUP_MSG_ID" "ZSS_LOG_PORT_OCCUP_MSG_TEXT
+
+/* MVD Server (Datasets) */
+
+#ifndef ZSS_LOG_DS_NAME_HLQ_MSG_ID
+#define ZSS_LOG_DS_NAME_HLQ_MSG_ID            LOG_MSG_PRFX"1038I"
+#endif
+#define ZSS_LOG_DS_NAME_HLQ_MSG_TEXT          "l1=%s\n"
+#define ZSS_LOG_DS_NAME_HLQ_MSG               ZSS_LOG_DS_NAME_HLQ_MSG_ID" "ZSS_LOG_DS_NAME_HLQ_MSG_TEXT
+
+#ifndef ZSS_LOG_DS_SERVING_NAME_MSG_ID
+#define ZSS_LOG_DS_SERVING_NAME_MSG_ID        LOG_MSG_PRFX"1039I"
+#endif
+#define ZSS_LOG_DS_SERVING_NAME_MSG_TEXT      "Serving: %s\n"
+#define ZSS_LOG_DS_SERVING_NAME               ZSS_LOG_DS_SERVING_NAME_MSG_ID" "ZSS_LOG_DS_SERVING_NAME_MSG_TEXT
```

Comment:
Some `#define`s have the _MSG suffix and some don't. For example, this one. Can you please add _MSG for consistency?